### ISSUE TITLE
Rethinking lists

### ISSUE THREAD

**Issue by shtrom** [pump-io/pump.io#562]
The concepts of lists is good, however I find it a bit limited at the moment.

First, shouldn't the followers and following groups also be treated as lists (they are in some cases)? Of course, the followers would be _added_ by the system, but it would be good if I could delete them. Same for who I am following. Granted, that's what the Unfollow button is for, but it seems to me it is two different concept for what is inherently the same thing. Following/unfollowing is fine in the rest of the context, but when displaying a list of them, we just want to remove them. Adding by webfinger would also save time.

Then, it would be nice to be able to choose which list(s) by default get
- in the To field for major eventsq
- in the Cc field for major events (I still don't quite get the difference between these two fields in the context of a broadcast system)q
- the minor event updates and,
- more generally access to which of my profile information (I don't really want people I have put into the “Annoying” list to know it, so I should be able to block access to my lists to these people).

In summary, the follow(ers|ing) list should behave just like any other lists and these lists should be usable to define access control policies.
